src/components/Notification/Notification.jsx

```diff
@@ -127,27 +127,22 @@ const StyledAside = styled.aside`
     border-top-width: ${props => props.theme.borderWidth};
   }
 
-  .icon {
-    display: inline-block;
-    color: ${props => props.theme.alert[props.context].borderColor};
-    background: transparent;
-    border-width: 0;
-    height: 100%;
-    margin-right: 1rem;
-    line-height: 4rem;
-  }
-
   .close-btn {
     display: inline-block;
-    color: ${props => props.theme.controlColor};
-    background: transparent;
-    border-width: 0;
-    height: 100%;
-    padding: 0 1rem;
-    margin-left: 0.5rem;
+    position: absolute;
```

Comment:
Could you verify the positioning of the cross ? It's not always correctly centered.

Reply:
It is fixed!